### DISCUSSION TITLE
Universal App: Higher priority Layout Route "./_layout.tsx" overriding redundant Layout Route "./_layout"

### DISCUSSION THREAD

**Discussion by seyaobey-dev** [expo/router#655] [Q&A]
I would like to build a universal using `expo-router + react-native-web`. When I try to create a production build with command 
`npx expo export:web`, I'm getting the error
 
```
CommandError: expo export:web can only be used with Webpack. Use expo export for other bundlers.
```

To fix this, I decide to remove the `"bundler": "metro"` from `app.json` (though this is required for `expo-router` to work). Now I can run the command and produce the build in the`web-build` folder. But now when I serve locally with `npx serve web-build` or I deploy to vercel, I'm getting a blank page with the following console error:
```
 Higher priority Layout Route "./_layout.tsx" overriding redundant Layout Route "./_layout". Remove the Layout Route "./_layout" to fix this.
```
Here is the github link to the project:
https://github.com/paygroup/paygroup.expo

**Comment by FreddzieAI** [comment]
same error here was hoping for a fix. did you manage to figure it out?
